src/main/scala/rocket/nbdcache.scala

```diff
@@ -6,11 +6,10 @@ import Chisel._
 import uncore.tilelink._
 import uncore.coherence._
 import uncore.agents._
-import uncore.util._
 import uncore.constants._
-import util.{ParameterizedBundle, DecoupledHelper}
+import uncore.util._
```

Comment:
I'm wondering if we should promote uncore/util/AMOALU.scala to util as well? Stuff that's used by both the L1 and L2 caches is a bit of a grey area...